### PR TITLE
MarkerData -->> DataTable

### PR DESCRIPTION
**Pull request by klshrinidhi**
Replace MarkerData class with DataTable in:
- MarkersReference
- MarkerPlacer

which are in turn used by IK and Scale.

Files to review:
- Applications/IK/test/testIK.cpp
- Applications/Scale/test/testScale.cpp
- OpenSim/Simulation/MarkersReference.cpp
- OpenSim/Simulation/MarkersReference.h
- OpenSim/Tools/InverseKinematicsTool.cpp
- OpenSim/Tools/MarkerPlacer.cpp
- OpenSim/Tools/MarkerPlacer.h

**The branch in #709 is merged into this branch to make data-adapters available for use. Consider merging #709 before merging this.**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -445,6 +445,11 @@ if(NOT SIMBODY_HOME AND OPENSIM_DEPENDENCIES_DIR)
 endif()
 
 
+## Find BTK.
+find_package(BTK REQUIRED)
+include(${BTK_USE_FILE})
```
Comment (chrisdembia) [inline CMakeLists.txt]: This can be removed, right? The lines above already have `find_package(BTK...`

**Comment by aseth1** [comment]
@klshrinidhi I apologize for letting this PR fall off the radar. Given the nice improvements you have made to DataTable/TimeSeriesTable I think we could either update this PR or submit a new one it its place. Being able to provide marker data as Tables would make doing IK from various sources (including synthetic data for testing) a lot easier. Let me know your thoughts on reviving this PR.

**Comment by aseth1** [comment]
This PR has been superseded by #1372.

**Comment by aseth1** [comment]
@klshrinidhi if you don't need this branch anymore can you delete it?

**Pull request closed without merging**